index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css">
+  <title>Home Page</title>
+</head>
+<body>
+  <header>
+    <div class="social">
+      <ul>
+        <li><a href=""><i class="fa-brands fa-facebook-f"></i></a></li>
+        <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
```

Comment:
- [ ] @Yacoubou-seidou great work implementing the changes requested, however, as previously requested kindly reduce the spacing between the social media icons 
![image](https://user-images.githubusercontent.com/61408860/225136479-222ad1c4-b2b5-43f1-900b-5908353a951b.png)